InferenceSystem/README.md

```diff
@@ -68,13 +74,34 @@ export AZURE_COSMOSDB_PRIMARY_KEY="<yourprimarykey>"
 
 ```
 cd InferenceSystem
-python LiveInferenceOrchestrator.py
+python src/LiveInferenceOrchestrator.py --config ./config/Test/FastAI_LiveHLS_OrcasoundLab.yml
 ```
 
-You should see a bunch of .wav and .pngs written to the storage account.
-You should see entries in the CosmosDB.
+You should see the following logs in your terminal. Since this is a Test config, no audio is uploaded to Azure and no metadata is writteb to CosmosDB.
```

Comment:
```suggestion
You should see the following logs in your terminal. Since this is a Test config, no audio is uploaded to Azure and no metadata is written to CosmosDB.
```